Changelog.md

```diff
@@ -8,10 +8,12 @@ Compiler Features:
  * EVM: Support for the EVM Version "Shanghai".
  * NatSpec: Add support for NatSpec documentation in ``enum`` definitions.
  * NatSpec: Add support for NatSpec documentation in ``struct`` definitions.
+ * Optimizer: Minimal Yul optimizations needed to prevent "Stack too deep" errors (i.e. the stack allocation optimization and the ``UnusedPruner`` optimizer step) are now performed even when the user requests Yul optimizer to be disabled.
```

Comment:
Too much detail. Just state that we run stack-to-memory by default whenever possible.

Reply:
Changed.

Actually, should I even keep it under "Optimizer" if we logically no longer consider it an optimization? "Code Generator"?